lib/omniauth/strategies/openid_connect.rb

```diff
@@ -344,8 +374,17 @@ def valid_response_type?
         false
       end
 
+      # 'code', ['id_token', 'token'], or [:id_token, :token]
```

Comment:
I think it's a bit confusing to pass in an array and a string. Better just to keep it as a string.